specification/archSpec/base/ditamap-attributes.dita

```diff
@@ -49,10 +49,7 @@
 						generate next/previous links for online presentation. This attribute is
 						available in topics on the <xmlelement>linklist</xmlelement> and
 							<xmlelement>linkpool</xmlelement> elements, where it has the same
-						behavior. Where the <xmlatt>collection-type</xmlatt> attribute is available
```

Comment:
Not sure this should be removed, because you can still set collection-type on elements that do not contain elements, and in those cases the behavior of the attribute is still undefined. I think rather than removing we should just rephrase to say that, and remove `reltable` as an example element.